test_util/azure_test_driver.py

```diff
@@ -123,70 +131,77 @@ def poll_deploy():
-        master_url = "http://{}".format(master_lb)
+        master_lb = deployment_result.properties.outputs['masterFQDN']['value']
 
         print("Template deployed using SSH private key: https://mesosphere.onelogin.com/notes/18444")
-        print("For troubleshooting, master0 can be reached using: ssh -p 2200 core@{}".format(master_lb))
-
-        @retry(wait_fixed=(5 * 1000), stop_max_delay=(15 * 60 * 1000))
-        def poll_on_dcos_ui_up():
-            r = get_dcos_ui(master_url)
-            assert r is not None and r.status_code == requests.codes.ok, \
-                "Unable to reach DC/OS UI: {}".format(master_url)
-
-        print("Waiting for DC/OS UI at: {} ...".format(master_url))
-        poll_on_dcos_ui_up()
+        print("For troubleshooting, master0 can be reached using: ssh -p 2200 {}@{}".format(
+            get_value('linuxAdminUsername'), master_lb))
 
         # Run test now, so grab IPs
         nmc = NetworkManagementClient(credentials, subscription_id)
         ip_buckets = {
-            'masterNodeNic': [],
-            'slavePrivateNic': [],
-            'slavePublicNic': []}
-
-        for resource in rmc.resource_groups.list_resources(group_name):
-            for bucket_name, bucket in ip_buckets.items():
-                if resource.name.startswith(bucket_name):
-                    nic = nmc.network_interfaces.get(group_name, resource.name)
-                    all_ips = []
-                    for config in nic.ip_configurations:
-                        all_ips.append(config.private_ip_address)
-                    bucket.extend(all_ips)
-
-        with closing(SSHTunnel('core', 'ssh_key', master_lb, port=2200)) as t:
+            'master': [],
+            'private': [],
+            'public': []}
+
+        for resource in rmc.resource_groups.list_resources(
+                group_name, filter=("resourceType eq 'Microsoft.Network/networkInterfaces' or "
+                                    "resourceType eq 'Microsoft.Compute/virtualMachineScaleSets'")):
+            if resource.type == 'Microsoft.Network/networkInterfaces':
+                nics = [nmc.network_interfaces.get(group_name, resource.name)]
+            elif resource.type == 'Microsoft.Compute/virtualMachineScaleSets':
+                nics = list(nmc.network_interfaces.list_virtual_machine_scale_set_network_interfaces(
+                            virtual_machine_scale_set_name=resource.name, resource_group_name=group_name))
+            else:
+                raise('Unexpected resourceType: {}'.format(resource.type))
+
+            for bucket_name in ip_buckets.keys():
+                if bucket_name in resource.name:
+                    for n in nics:
+                        for config in n.ip_configurations:
+                            ip_buckets[bucket_name].append(config.private_ip_address)
+
+        print('Detected IP configuration: {}'.format(ip_buckets))
```

Comment:
more of a nit than anything, I see the rest of this script uses print() so this is consistent.